x/bank/client/cli/sendtx.go

```diff
@@ -24,7 +21,7 @@ const (
 // SendTxCmd will create a send tx and sign it with the given key.
 func SendTxCmd(cdc *codec.Codec) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "send",
+		Use:   "send [to_address] [amount]",
 		Short: "Create and sign a send tx",
 		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
I think you need to drop `ExactArgs` around here

Reply:
Done.